### ISSUE TITLE
include name of sketch when asking user “Save sketch before closing?” 

### ISSUE THREAD

**Issue by StanLepunK** [processing/processing#3418]
Suggestion for the managing of the sketch unsaved when you close Processing. 
May be that's be better if when you close Processing, Processing return the name of the file is not saved and ask if the user want save it, and after go to the next file unsaved.
![capture d ecran 2015-06-20 a 16 27 36](https://cloud.githubusercontent.com/assets/3332269/8279543/d1667820-18d5-11e5-95ae-7f4e8d64c6ec.png)

**Comment by benfry** [comment]
Processing already asks if you want to save. Are you saying that the dialog box should include the name of the sketch?

**Comment by StanLepunK** [comment]
Exactly, I think that's can be good that the dialog box include the name of the sketch. Sorry for my english.

**Comment by joshgiesbrecht** [comment]
Is this specific to OS X?  I just checked on Windows and it already shows the name of the sketch when asking if you want to save.

**Comment by benfry** [comment]
OS X uses different code there in order to follow OS X HIG guidelines, but both needed to be updated slightly so that they could be localized properly. Will require updates for the individual language .properties files:
```
save.title = Do you want to save changes to %s<br> before closing?
close.unsaved_changes = Save changes to %s?
```

Fixed for 3.0 beta 3.
